thunder/benchmarks/litgpt_chunks.py

```diff
@@ -0,0 +1,242 @@
+from dataclasses import dataclass
+from functools import wraps
+from itertools import groupby, product
+from collections.abc import Callable, Sequence
+
+import pytest
+
+import torch
+
+from litgpt import Config, GPT
+from litgpt.config import configs
+
+import thunder
+
+from thunder.benchmarks import (
+    inductor_cutlass_executor,
+    inductor_triton_executor,
+    thunder_executor,
+    torch_compile_executor,
+    torch_executor,
+)
+from thunder.common import CompileData, CompileStats
+from thunder.core.compile_data import set_compile_data_and_stats
+
+from thunder.core.jit_ext import thunder_general_jit
+from thunder.core.langctxs import set_langctx
+from thunder.core.trace import TraceCtx
+from thunder.core.transforms import eval_trace
+from thunder.executors.torch_compile import to_torch_translator
+
+from thunder.tests.make_tensor import make_tensor
+
+BATCH_SIZE = 2
+CONFIG_NAMES = list(sorted(c["name"] for c in configs))
```

Comment:
You could keep only the configs that have distinct values in https://github.com/Lightning-AI/litgpt/blob/78bd4cae1e655359e92e7c7b830fcbaa4c15c152/litgpt/config.py#L17 (excluding `name` and `hf_config`). Otherwise you'll be running the same archs more than once